app/models/user.py

```diff
@@ -225,7 +225,13 @@ def has_permissions(self, *permissions, restrict_admin_usage=False, allow_org_us
         if not service_id and not org_id:
             # we shouldn't have any pages that require permissions, but don't specify a service or organisation.
             # use @user_is_platform_admin for platform admin only pages
-            raise NotImplementedError
+            # raise NotImplementedError
+
+            # We now have a use case where even a user isn't a part of a service
```

Comment:
I'm not sure that this can happen currently unless the user is created outside the UI. If that's correct, can this stay as it was?

If it should change, let's also change the first line of the comment above since they would conflict

Reply:
Thanks for catching this, @stvnrlly!

This came about because of using the other decorator previously.  Perhaps with just the login decorator we've switched to now, we don't need to make this change.

I'll adjust this and update as needed!

Reply:
Yup, no longer needed - I reverted the changes!